web/pandas/index.html

```diff
@@ -0,0 +1,157 @@
+                        </ul>
+                    </div>
+                    <div class="col-md-4">
+                        <h5>Documentation</h5>
+                        <ul>
+                            <li><a href="#">User guide</a></li>
+                            <li><a href="#">API reference</a></li>
+                            <li><a href="#">Tutorials</a></li>
+                            <li><a href="#">Release notes</a></li>
+                        </ul>
+                    </div>
+                    <div class="col-md-4">
+                        <h5>Community</h5>
+                        <ul>
+                            <li><a href="{{ base_url }}/community/about.html">About pandas</a></li>
+                            <li><a href="https://stackoverflow.com/questions/tagged/pandas">Ask a question</a></li>
+                            <li><a href="{{ base_url }}/community/ecosystem.html">Ecosystem</a></li>
+                        </ul>
+                    </div>
+                </div>
+                <section>
+                    <h5>With the support of:</h5>
+                    <div class="row h-100">
+                        {% for company in partners.active %}
+                            <div class="col-md-2 my-auto">
+                                <a href="{{ company.url }}" target="_blank">
+                                    <img class="img-fluid" alt="{{ company.name }}" src="{{ base_url }}{{ company.logo }}"/>
+                                </a>
+                            </div>
+                        {% endfor %}
+                    </div>
+                </section>
+            </div>
+            <div class="col-md-3">
+                {% if releases %}
+                    <h4>Latest version: {{ releases[0].name }}</h4>
+                    <ul>
+                        <li><a href="https://pandas.pydata.org/pandas-docs/stable/whatsnew/v0.25.0.html">What's new in {{ releases[0].name }}</a></li>
+                        <li>Release date:<br/>{{ releases[0].published.strftime("%b %d, %Y") }}</li>
+                        <li><a href="https://pandas.pydata.org/pandas-docs/stable/">Documentation (web)</a></li>
+                        <li><a href="https://pandas.pydata.org/pandas-docs/stable/pandas.pdf">Documentation (pdf)</a></li>
+                        <li><a href="{{ releases[0].url }}">Download source code</a></li>
+                    </ul>
+                {% endif %}
+                <h4>Follow us</h4>
+                <div class="text-center">
+                    <p>
+                        <a href="https://twitter.com/pandas_dev?ref_src=twsrc%5Etfw" class="twitter-follow-button" data-show-count="false">Follow @pandas_dev</a><script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+                    </p>
+                </div>
+				<h4>Get the book</h4>
+				<p>
+					<a href="http://shop.oreilly.com/product/0636920050896.do">
```

Comment:
If you want to change this to an affiliate link (so we receive ad fees), here is the one: https://amzn.to/2KI5JJw. I can donate annual proceeds from this link (probably not a lot in aggregate) to NumFOCUS

Reply:
That makes sense, thank you. Updated.